MERN/MERN_CRUD/server.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+
+const app = express();
+
+//import routes
+const postRoutes = require("./routes/posts");
+
+//app middleware
+app.use(bodyParser.json());
+app.use(cors());
+
+//route middleware
+app.use(postRoutes);
+
+const PORT = 8000;
+
+const DB_URL =
+  "mongodb+srv://nisal:nisal@mernapp.wopdx.mongodb.net/mernCrud?retryWrites=true&w=majority";
```

Comment:
Replace connection string with a place holder.
The use of a .env file would be good too I think, but will need instructions on how to set it up.